tests/test_contexts.py

```diff
@@ -26,6 +26,14 @@ def test_xenonnt_online():
     st.search_field('time')
 
 
+def test_xennonnt(cmt_version='ONLINE'):
+    st = xenonnt(_database_init=False, use_rucio=False)
+    st.search_field('time')
+
+def test_xennonnt_v1(cmt_version = 'v1'):

```

Comment:
Yes, this won't work because old_global version have not a complete set of corrections as the latest one, we can check if global_v1 contains all corrections, if not, do not set the "local_version" and let the default value